### PR TITLE
Upgrade AirMap support to 2.0.1 to allow building for ARM

### PR DESCRIPTION
**Pull request by remster**
This addresses: https://github.com/airmap/platform-sdk/issues/182
and thus allows building QGC with AirMap support for ARM

### CODE REVIEW

**Comment by petronny** [comment]
I confirm that this PR works when compiling qgroundcontrol with airmap support on archlinuxarm.

File: src/Airmap/AirMapAdvisoryManager.cc [hunk 1]
```diff
@@ -74,7 +74,6 @@ AirMapAdvisoryManager::_requestAdvisories()
     _advisories.clearAndDeleteContents();
 
     Advisory::Search::Parameters params;
-    params.authorization = _shared.loginToken().toStdString();
```
Comment (remster) [inline src/Airmap/AirMapAdvisoryManager.cc:77]: this is a bit of a mystery for me; advisory.h [never had](https://github.com/airmap/platform-sdk/commits/master/include/airmap/advisory.h) an `authorization` member in its `Advisory::Search::Parameters`

File: src/Airmap/AirMapRulesetsManager.cc [hunk 2]
```diff
@@ -247,7 +247,6 @@ void AirMapRulesetsManager::setROI(const QGCGeoBoundingCube& roi, bool reset)
     _ruleSets.clearAndDeleteContents();
     _state = State::RetrieveItems;
     RuleSets::Search::Parameters params;
-    params.authorization = _shared.loginToken().toStdString();
```
Comment (remster) [inline src/Airmap/AirMapRulesetsManager.cc:250]: [same](https://github.com/airmap/platform-sdk/commits/master/include/airmap/rulesets.h) with `rulesets.h`

**Comment by booo** [comment]
Please use meaning full commit messages and add a proper commit body to describe your changes. Messages such as "migrate" or "fix upstream" do not help at all to understand what you did.

**Comment by remster** [comment]
> Please use meaning full commit messages and add a proper commit body to describe your changes. Messages such as "migrate" or "fix upstream" do not help at all to understand what you did.

Sorry about this - commits enriched (hopefully) to requested standards

**Comment by remster** [comment]
@booo a gentle reminder about this getting stale

**Comment by booo** [comment]
Thanks for the changes. Unfortunately I don't have commit/merge rights. I hope someone from the dev team can take a look soon.

**Review by patrickelectric — APPROVED** [review]
Great job

**Comment by farhangnaderi** [comment]
@remster thanks a bunch for making this happen. However as I enable Airmap through json file in Airmap folder I can only see it available on Linux Desktop version. On android no Airmap tab appears. Could you please guide me if I am missing a step here?

WhatsApp Image 2023-03-31 at 00 12 27

**Pull request merged**